packages/gp2-components/src/organisms/ResourceModal.tsx

```diff
@@ -0,0 +1,158 @@
+import { gp2 } from '@asap-hub/model';
+import {
+  Button,
+  EditModal,
+  Headline3,
+  LabeledDropdown,
+  LabeledTextField,
+  Link,
+  Paragraph,
+  pixels,
+  utils,
+} from '@asap-hub/react-components';
+import { UrlExpression } from '@asap-hub/validation';
+import { css } from '@emotion/react';
+import { useState } from 'react';
+import { mobileQuery } from '../layout';
+
+const types = ['Link' as const, 'Note' as const];
```

Comment:
we should probably use the types from the model

Reply:
Done